resolvers.js

```diff
@@ -526,6 +542,27 @@ const Resolvers = {
         logger.error(e.message);
         return e.message;
       }
+    },
+
+    async deleteOpticalImage(_, args) {
+      let {datasetId} = args;
+      const payload = jwt.decode(args.jwt, config.jwt.secret);
+      try {
+        logger.info(args);
```

Comment:
Maybe worth skipping this, or at least printing requested dataset ID instead of the whole payload